<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userdatadialog</class>
 <widget class="QWidget" name="userdatadialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Renseigner vos données utilisateur</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="lbFistname">
     <property name="text">
      <string>Prénom</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leFisrtname"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbLastname">
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbCompany">
     <property name="text">
      <string>Société</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="leLastname"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="leCompany"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lePhone"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lbEmail">
     <property name="text">
      <string>Mail</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lbPhone">
     <property name="text">
      <string>Tél. fixe</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="leEmail"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Adresse</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbAddress">
        <property name="text">
         <string>N°, Rue</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbPostalCode">
        <property name="text">
         <string>Code Postal</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lePostalCode"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lbCity">
        <property name="text">
         <string>Ville</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="leCity"/>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="leAddress"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Pays</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="leMobilePhone"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lbSiret">
     <property name="text">
      <string>N°Siret</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="leSiret"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lbMobilePhone">
     <property name="text">
      <string>Tél. mobile</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnValid">
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
